tests/tests/fixtures/tables/partitioned.rs

```diff
@@ -29,28 +29,35 @@ pub struct PartitionedTable {
 
 impl PartitionedTable {
     pub fn setup() -> String {
-        PARTITIONED_TABLE_SETUP.into()
+        Self::setup_with_indexed_columns(["description", "sale_date", "amount"])
     }
-}
 
-static PARTITIONED_TABLE_SETUP: &str = r#"
-BEGIN;
-    CREATE TABLE sales (
-        id SERIAL,
-        sale_date DATE NOT NULL,
-        amount REAL NOT NULL,
-        description TEXT,
-        PRIMARY KEY (id, sale_date)
-    ) PARTITION BY RANGE (sale_date);
+    pub fn setup_with_indexed_columns<'a>(
+        indexed_columns: impl IntoIterator<Item = &'a str>,
+    ) -> String {
+        let formatted_indexed_columns = indexed_columns.into_iter().collect::<Vec<_>>().join(", ");
+        format!(
+            r#"
+            BEGIN;
+                CREATE TABLE sales (
+                    id SERIAL,
+                    sale_date DATE NOT NULL,
+                    amount REAL NOT NULL,
+                    description TEXT,
+                    PRIMARY KEY (id, sale_date)
+                ) PARTITION BY RANGE (sale_date);
 
-    CREATE TABLE sales_2023_q1 PARTITION OF sales
-      FOR VALUES FROM ('2023-01-01') TO ('2023-03-31');
+                CREATE TABLE sales_2023_q1 PARTITION OF sales
```

Comment:
sorry for the noise, but why all the whitespace changes?

(I'll do a proper review on monday)

Reply:
If you're referring to the changes in `PdbScan`, please see https://github.com/paradedb/paradedb/pull/2382#issuecomment-2764776520

...if to something else, then maybe because this PR is a dependent PR atop #2370? It might not need to be though: I'm not sure that this PR has anything to do with partitioning, in which case I can detach it.

Reply:
Ah, I see. In this file it was due to the need to template which columns were indexed. As to whitespace in `PdbScan`: see https://github.com/paradedb/paradedb/pull/2382#issuecomment-2764776520